Common/Currency.swift

```diff
@@ -60,14 +60,20 @@ public extension Currency {
     static let INR = Currency(name: "Indian rupee", code: "INR")
     static let BRL = Currency(name: "Brazilian real", code: "BRL")
     static let ZAR = Currency(name: "South African rand", code: "ZAR")
-    
     static let Bitcoin = Currency(name: "Bitcoin", code: "BTC")
     static let Ethereum = Currency(name: "Ethereum", code: "ETH")
     static let Litecoin = Currency(name: "Litecoin", code: "LTC")
     static let Ripple = Currency(name: "Ripple", code: "XRP")
     static let Cardano = Currency(name: "Cardano", code: "ADA")
     static let NEM = Currency(name: "NEM", code: "XEM")
     static let USDT = Currency(name: "Tether USD", code: "USDT")
+    static let ETC = Currency(name: "Ethereum Classic", code: "ETC")
+    static let BCH = Currency(name: "Bitcoin Cash", code: "BCH")
+    static let DOGE = Currency(name: "Dogecoin", code: "DOGE")
```

Comment:
Could you please remove these altcoin currency static members, since I've already added several of them in 0.0.4? Also, I would like to name them using the name instead of their code. Like `bitcoin` instead of `BTC`. (I know USDT is a typo there. Will rename it as USDTether in future release, Also, NEM is an exception there)